models/bamf_pet_ct_breast_tumor/config/default.yml

```diff
@@ -0,0 +1,49 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Breast FDG-avid lesions segmentation (dicom to dicom)
+
+execute:
+- FileStructureImporter
+- SitkNiftiConverter
+- Registration
+- NNUnetPETCTRunner
+- TotalSegmentatorMLRunner
+- BreastPostProcessor
+- DsegConverter
+- DataOrganizer
+
+modules:
+  FileStructureImporter:
+    input_dir: 'input_data'
+    structures:
+      - $patientID@instance/ct@dicom:mod=ct
+      - $patientID/pt@dicom:mod=pt
+    import_id: patientID
+
+  SitkNiftiConverter:    
+    in_datas: dicom:mod=pt|ct
+    allow_multi_input: true
+
+  NNUnetPETCTRunner:
+    in_ct_data: nifti:mod=ct:registered=true
+    nnunet_task: Task762_PET_CT_Breast
+    nnunet_model: 3d_fullres
+    roi: LIVER,KIDNEY,URINARY_BLADDER,SPLEEN,LUNG,BRAIN,HEART,STOMACH,BREAST+FDG_AVID_TUMOR
+
+  TotalSegmentatorMLRunner:
+    in_data: nifti:mod=ct:registered=true
+    use_fast_mode: false
```

Comment:
See above.

Reply:
I'd assume that fast mode would work just as well but should significantly speed up the process.